src/utils.py

```diff
@@ -310,3 +312,47 @@ def get_response_data(json_response, data_default):
     Data default parameters takes what should data to be returned.
     """
     return json_response.get("result", {}).get("data", data_default)
+
+
+def server_run_flow(flow_name, flow_args):
```

Comment:
this needs to be checked (and I know, it is complicated :/)

Reply:
Could you elaborate what needs to be checked here? @tisnik